benchmarks/cassandra-stress/params.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kudo.dev/v1beta1
+parameters:
+  - name: TARGET_HOST
+    description: "A cassandra node that is used by cassandra stress"
+    required: true
+
+  - name: PARALLEL
+    description: "Number of Cassandra Stress instances to start"
+    default: "3"
+
+  - name: THREADS_PER_CLIENT
+    description: "Number of threads per client"
```

Comment:
Nit: What's a client in this context? Is there a client per instance? Should this be `THREADS_PER_INSTANCE`?

Reply:
I've adjusted the `PARALLEL` parameter to say `NUM_CLIENTS` and changed the docs, should be more understandable now